FIPS/config.md

```diff
@@ -0,0 +1,499 @@
+|p2pSeed|MainSeedURL<br>TestSeedURL<br>LocalSeedURL<br>CustomSeedURL|N/A|Rolling into one setting with new group system|
+|p2pFanout|N/A|broadcastnum||
+|p2pSpecialPeers|MainSpecialPeers<br>TestSpecialPeers<br>LocalSpecialPeers<br>CustomSpecialPeers|peers|Rolling into one setting with new group system|
+|p2pConnectionPolicy|N/A|exclusive<br>exclusive_in|"exclusive_in" was a more powerful setting of "exclusive", so those two booleans have been rolled into three connection policies: NORMAL,ACCEPT,REFUSE|
+|p2pTimeout|N/A|deadline|Name is up for debate but I believe "timeout" gives people a better understanding even though "deadline" is more technically accurate|
+|logLevel|logLevel|loglvl||
+|logPath|LogPath|N/A||
+|logJSON|N/A|logjson||
+|logLogstash|N/A|logstash<br>logurl|This was a tandem setting that has been rolled into one. If there's a URL present, it's also enabled|
+|logStdOut|N/A|stdoutlog|Grouping|
+|logStdErr|N/A|stderrlog|Grouping|
+|logMessages|N/A|debuglog|More descriptive name|
+|logDBStates|N/A|wrproc|wrproc was short for "write processed db states"|
+|simNoInput|N/A|sim_stdin|Grouping + Name change to reflect default of false|
+|simCount|N/A|cnt|Grouping|
+|simFocus|N/A|node|Grouping + more descriptive name|
+|simNet|N/A|net|Grouping|
+|simNetFile|N/A|fnet|Grouping|
+|simDropRate|N/A|drop|Grouping + more descriptive name|
+|simTimeOffset|N/A|timedelta|Grouping. Name change up for debate|
+|simRuntimeLog|N/A|runtimelog|Grouping|
+|simWait|N/A|waitentries|Grouping|
+|debugConsole<br>debugConsolePort|N/A|debugconsole|This has been split up into two settings. Previously it was `(localhost|remote):<port>`. `localhost|remote` or empty setting is now `debugConsole` of OFF, LOCAL, ON, with the port specified separately.|
+|chainHeadFix|N/A|checkheads<br>fixheads|Two booleans with only 3 valid settings rolled into one with: OFF, IGNORE, ON|
+|oneLeader|N/A|rotate|Match the name in the code|
+|keepMismatch|N/A|keepmismatch||
+|forceSync2Height|N/A|sync2|More descriptive|
+|journalFile<br>journalMode<br>journalType|N/A|journaling<br>journal<br>follower<br>leader|Before, "journaling" turned it on and off, "journal" specified a file. If the file did not exist it was writing, if the file existed it was reading. There were also two booleans for forcing followers and leader, where (true, true) is invalid.|
+|pluginPath|N/A|plugin|More descriptive|
+|pluginTorrent<br>pluginTorrentUpload|N/A|tormanage<br>torupload||
+
+### Deprecated Settings
+
+|Config | Command Line | Reason |
+|-------|--------------|--------|
+|FastBootLocation||This still works but there's some inconsistency with regards to simulated nodes, which overwrite this setting. There's no reason to save this anywhere other than homedir, which is the default, but if this setting is still desired I'm open to un-deprecating it |
+|ControlPanelFilesPath||Exists but has no effect|
```

Comment:
This was prior to compiling in the control panel source code. Now it is all included in the binary, so completely useless now :+1: